github/github.go

```diff
@@ -72,6 +72,9 @@ const (
 
 	// https://developer.github.com/changes/2016-04-01-squash-api-preview/
 	mediaTypeSquashPreview = "application/vnd.github.polaris-preview+json"
+
+	// https://developer.github.com/changes/2016-04-04-git-signing-api-preview/
+	mediaTypeGitSigningPreview = "application/vnd.github.cryptographer-preview"
```

Comment:
Shouldn't you include the `+json` suffix here?

As far as I understand, we want to use the default mediaTypeV3 value of `application/vnd.github.v3+json`, except to replace the version `v3` with `cryptographer-preview`.

See https://developer.github.com/v3/media/ for explanation of +json part of media type.